sdk/eventhubs/microsoft-azure-eventhubs-eph/pom.xml

```diff
@@ -14,14 +14,10 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.microsoft.azure</groupId>
   <artifactId>azure-eventhubs-eph</artifactId>
-  <version>3.3.0-beta.1</version> <!-- {x-version-update;com.microsoft.azure:azure-eventhubs-eph;current} -->
+  <version>3.2.3</version> <!-- {x-version-update;com.microsoft.azure:azure-eventhubs-eph;current} -->
 
   <name>Microsoft Azure SDK for Event Hubs Event Processor Host(EPH)</name>
-  <description>EPH is built on top of the Azure Event Hubs Client and provides a number of features not present in that lower layer.${line.separator}
-    This library has been replaced by new Azure SDKs, you can read about them at https://aka.ms/azsdkvalueprop. The latest libraries to interact with the Azure Event Hubs service are:${line.separator}
-    1. https://search.maven.org/artifact/com.azure/azure-messaging-eventhubs${line.separator}
-    2. https://search.maven.org/artifact/com.azure/azure-messaging-eventhubs-checkpointstore-blob${line.separator}
-    It is recommended that you move to the new package.
```

Comment:
Looks like a similar note existed before but it was missing the migration guide. It might be good to include the links to the new artifacts if we are updating the description.

Reply:
Yes, there was a note before, but it was not following the standard text that we are now using across the board. How about this for the link update: https://github.com/Azure/azure-sdk-for-java/pull/21103/files#r625345332

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-java/pull/21103/commits/40371562d6f2ad295f70c3c2c28569c94172a827